joomla/joomla-cms
# [languages] Organize and new metadata (Native language and calendar type)

### Summary of Changes
1. Add `nativeName` and `calendar` to language metadata xml.

This new `nativeName` metadata is the language title in the native language.
This should be added to all languages packages.

For consistence i added in the 3 language metadata xmls.

After this is merged it can be used in installation and when installing languages.

This new `calendar` metadata is the calendar type to be used in the new datapicker.
Example: gregorian, jalali
1. Uniformized all en-GB language title to be:
2. title: English (en-GB)
3. native title: English (United Kingdom)
### Testing Instructions
1. Code review.
2. Apply this patch, delete configuration.php, refresh page and install Joomla 
3. Check the english title is "English (en-GB)" and the native title is "English (United Kingdom)" in Extensions -> Languages -> Content Languages
4. Check the english title is "English (en-GB)" in Extensions -> Languages -> Installed Languages in Both Site and Admin
5. Check the names of the language extensions in Extensions -> Manage -> Manage (check language and package type)

Reggarding the unit test changes if they pass is ok.
### Documentation Changes Required

Pages that indicate how to create a language package need to be updated.

---

@brianteeman please check if the English native title is correct.
@infograf768 @Bakual After this is merged this new var can be used in installation and when installing content languages.
